ghost/core/core/server/web/parent/middleware/index.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
     emitEvents: require('./emit-events'),
     ghostLocals: require('./ghost-locals'),
     logRequest: require('./log-request'),
+    queueRequest: require('./queue-request'),
```

Comment:
We use `requestQueue` and `queueRequest`. We don't have a great standard given the other mw values. Could we be consistent about using `requestQueue`?

Reply:
heh, i actually opted for `queueRequest` in this situation as the convention for the other middleware was `<verb>Something` (`logRequest`, `emitEvents`) etc.